src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -4176,25 +4176,25 @@
     <value>AssemblyName.GetAssemblyName() is not supported on this platform.</value>
   </data>
   <data name="PlatformNotSupported_HybridGlobalizationWithCompareOptions" xml:space="preserve">
-    <value>CompareOptions = {0} are not supported when HybridGlobalization=true on this platform. Disable it to load larger ICU bundle, then use this option.</value>
+    <value>CompareOptions = {0} are not supported on Apple mobile platforms. Adjust your code or open an issue on https://github.com/dotnet/runtime if this is impacting your use case.</value>
```

Comment:
```suggestion
    <value>CompareOptions = {0} are not supported on Apple mobile platforms.</value>
```
I think this is sufficient. I do not think we have any other error message where we explicitly tell people to open issue in dotnet/runtime.